op-conductor/metrics/metrics.go

```diff
@@ -108,6 +110,11 @@ func NewMetrics() *Metrics {
 			Help:      "Time (in seconds) to execute conductor loop iteration",
 			Buckets:   []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10},
 		}),
+		rollupBoostConnectionAttempts: factory.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Can we have another Gauge metric tracking the count of clients connected to the websocket server (Hub) at any point in time?